README.md

```diff
@@ -334,6 +335,58 @@ Griffin is growing!  We're always looking for contributors and maintainers to he
 
 As Griffin continues to grow, we may also refactor into more of a plugin-based architecture so you only have to install what you need.
 
+# ⬆️ Migrating JSON-based configuration
+Older versions of Griffin used `JSON`-based configuration.  We've since changed the standard format to `YAML`.  Migrating this configration is fairly straightforward;  simply follow the steps below
```

Comment:
Isn't JSON valid YAML?  I wonder if we could just rename `griffin-config.{env}.json` to `griffin-config.{env}.yaml` and let the CLI do the rest next time it writes.  Might be worth testing.

Reply:
definitely possible, but given that this isn't likely widely used, I thought it would be better to have users migrate their configurations on their own

Reply:
Fair point, though it I do find it a bit odd using another language to convert the yaml.  Another option is creating a migrate command (e.g. `griffin migrate`) that can convert the JSON to YAML.  It might not be completely useless afterwards either as it could be used if more breaking changes are introduced 🤔 

I doubt I have access to your forked copy or I'd see if I could throw something together.

Reply:
>Fair point, though it I do find it a bit odd using another language to convert the yaml

it's definitely possible to write the conversion script in any language.  My thinking is that users will usually be ops-centric people who have more familiarity with python instead of JS/TS.

>I doubt I have access to your forked copy or I'd see if I could throw something together.

is there anything left with this change?  Seems mergeable as-is to me.  I've addressed every piece of feedback in the PR and been responsive to feedback.  If adding a `griffing migrate-config` command is desired, I think that's a fair feature addition to add later.

Reply:
if you'd like a JS script, ChatGPT says this works.  it requires the installation of the `js-yaml` package.

```js
const fs = require('fs');
const path = require('path');
const { execSync } = require('child_process');
const yaml = require('js-yaml');

function migrate(jsonPath) {
    const jsonConfig = JSON.parse(fs.readFileSync(jsonPath, 'utf8'));

    const yamlPath = jsonPath.replace('.json', '.yaml');
    const yamlConfig = yaml.safeDump(jsonConfig);
    fs.writeFileSync(yamlPath, yamlConfig);
}

const here = __dirname;
const environments = fs.readdirSync(here).filter(file => file.endsWith('.json'));

environments.forEach(env => {
    const envPath = path.join(here, env);
    migrate(envPath);

    // Remove the old json config
    execSync(`git rm ${envPath}`);
});
```